scipy/stats/_multivariate.py

```diff
@@ -960,6 +964,36 @@ def rvs(self, mean=None, rowcov=1, colcov=1, size=1, random_state=None):
             out = out.reshape(mean.shape)
         return out
 
+    def equivalent_multivariate_normal(self, mean=None, rowcov=1, colcov=1,
+                                       random_state=None):
```

Comment:
I have to agree with @ev-br's comment. People are going to somehow find this function and use it 'for real' if it's kept in the public scipy interface.